lib/adagrams.rb

```diff
@@ -0,0 +1,139 @@
+require 'csv'
+
+def return_letter_pool_hash
+  
+  letter_pool_hash = {
```

Comment:
This is great just as it is! I just wanted to mention for the future that this is usually a great example of when a global variable (ie. `LETTER_POOL_HASH = ...`). That way you wouldn't need to write a method and then gets called several times and a new copy of the hash wouldn't get created each time.